app/src/main/java/com/amaze/filemanager/utils/files/FileUtils.java

```diff
@@ -116,6 +116,26 @@ public static long folderSize(File directory, OnProgressUpdate<Long> updateState
         return length;
     }
 
+    //llrraa
```

Comment:
Please add your name to the license on the top of the file instead.

Reply:
ok.